includes/class-wc-stripe-intent-controller.php

```diff
@@ -298,8 +298,10 @@ public function create_payment_intent_ajax() {
 	 * @return array
 	 */
 	public function create_payment_intent( $order_id = null ) {
-		$amount = WC()->cart->get_total( false );
-		$order  = wc_get_order( $order_id );
+		$settings = get_option( 'woocommerce_stripe_settings', [] );
```

Comment:
Really we should get the option from the gateway with `$gateway->get_option( 'capture' );` instead of accessing other options directly.